packages/analytics-js/src/components/capabilitiesManager/CapabilitiesManager.ts

```diff
@@ -122,17 +130,13 @@ class CapabilitiesManager implements ICapabilitiesManager {
       // TODO: check if polyfill has been evaluated via polling or
       //  with the callback param in its url and an exposed function
       const onPolyfillLoad = (scriptId?: string) => Boolean(scriptId) && this.onReady();
-      this.externalSrcLoader
-        ?.loadJSFile({
-          url: state.loadOptions.value.polyfillURL ?? POLYFILL_URL,
-          id: 'rudderstackPolyfill',
-          async: true,
-          timeout: POLYFILL_LOAD_TIMEOUT,
-          callback: onPolyfillLoad,
-        })
-        .catch(e => {
-          this.onError(e);
```

Comment:
is the error handled internaly by externalSrcLoader loadJSFile method?

Reply:
Yes. This is similar to the HTTP client module.